### PR TITLE
[Edit]Python: Regular Expressions

### PR DESCRIPTION
**Pull request by caupolicandiaz**
### Description

This PR makes a number of corrections and enhancements including:
- Correcting missing capitalizations
- Bolding the first entry in several instances
- Codebyte examples incorrectly formatted as standard code blocks
- Adding a codebyte
- Other small formatting edits

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x ] All writings are my own.
- [x ] My entry follows the Codecademy Docs style guide.
- [x ] My changes generate no new warnings.
- [x ] I have performed a self-review of my own writing and code.
- [x ] I have checked my entry and corrected any misspellings.
- [x ] I have made corresponding changes to the documentation if needed.
- [x ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/python/concepts/regex/terms/split/split.md [hunk 1]
```diff
@@ -34,13 +34,36 @@ The other arguments include:
...
 
 ```py
 import re
 
-race_info = '''The top three race finishers were: Jane Smith (2:14), Sarah Long (2:18) and Suzy Reynolds (2:20).'''
+text = '**Note:** This method only takes positive arguments'
 
-re.split(r"\(|\)", race_info)
+print(re.split(r"\*\*|\:", text))
 # A backslash indicates that the parentheses are part of the pattern
```
Comment (SSwiniarski) [inline content/python/concepts/regex/terms/split/split.md:45]:
```suggestion
# A backslash indicates that the asterisks are part of the pattern
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@caupolicandiaz Looks good. I only had one suggestion to update a code comment.

**Comment by caupolicandiaz** [comment]
@SSwiniarski Good catch! I've made the edit.

**Review by SSwiniarski — APPROVED** [review]
@caupolicandiaz Looks good!

**Review by KTom101 — APPROVED** [review]
@caupolicandiaz LGTM 👍

**Comment by KTom101** [comment]
Links to edited files: [Regular Expressions](https://www.codecademy.com/resources/docs/python/regex), [re.findall](https://www.codecademy.com/resources/docs/python/regex/findall), [re.match](https://www.codecademy.com/resources/docs/python/regex/match), [re.search](https://www.codecademy.com/resources/docs/python/regex/search), [re.split()](https://www.codecademy.com/resources/docs/python/regex/split)

**Pull request merged**